core/controllers/contributor_dashboard.py

```diff
@@ -387,6 +386,21 @@ def _get_reviewable_exploration_opportunity_summaries(
             topic_exp_ids = (
                 topic_services.get_all_published_story_exploration_ids(
                     topic_id=topic.id))
+        in_review_suggestion_target_ids = (
+            suggestion_services
+            .get_reviewable_translation_suggestion_target_ids(
+                user_id, language
+            )
+        )
+        topic_exp_ids_targetted_by_in_review_suggestions = [
+            exp_id
+            for exp_id in topic_exp_ids
+            if exp_id in in_review_suggestion_target_ids
+        ]
+
+        pinned_opportunity_summary = None
+        if topic_name:
+            topic = topic_fetchers.get_topic_by_name(topic_name)
```

Comment:
This seems to duplicate the call in line 382. Is it worth doing the fetch just once at the outset for both those use cases?

Reply:
Ah. Thanks for pointing this out. I'll change the code so that we only fetch the topic with `topic_name` once.

Reply:
Done.